src/wp-includes/blocks/latest-posts.php

```diff
@@ -51,7 +51,7 @@ function render_block_core_latest_posts( $attributes ) {
 	$filter_latest_posts_excerpt_more = static function( $more ) use ( $attributes ) {
 		$use_excerpt = 'excerpt' === $attributes['displayPostContentRadio'];
 		/* translators: %1$s is a URL to a post, excerpt truncation character, default … */
-		return $use_excerpt ? sprintf( __( ' … <a href="%1$s" rel="noopener noreferrer">Read more</a>' ), esc_url( get_permalink() ) ) : $more;
+		return $use_excerpt ? sprintf( __( '… <a href="%1$s" rel="noopener noreferrer">Read more</a>' ), esc_url( get_permalink() ) ) : $more;
```

Comment:
@mujuonly Default WP also add space before after the excerpt content 🤔

Reply:
<img width="730" alt="Screenshot 2023-09-20 at 3 05 41 PM" src="https://github.com/WordPress/wordpress-develop/assets/459367/849f1ad3-e79b-46ac-86d9-cdf6cc1f3800">
:man-shrugging:

Reply:
I'm tempted to say that this should actually be:

```php
if ( $use_excerpt ) {
	return sprintf(
		' [&hellip;] ' . '<a href="%1$s" rel="noopener noreferrer">%2$s</a>',
		esc_url( get_permalink() ),
		__( 'Read more' )
	);
} else {
	return $more;
}
```
- Translators should not have to deal with markup in the string to be translated.
- This retains the space before and after the `...`.
- The `'Read more'` string [is already translated](https://github.com/costdev/wordpress-develop/blob/2a8f8eca77b7c54ce2a3f04a37c064ae7f8a6552/src/wp-includes/blocks/read-more.php#L37), so we should leverage this.

Reply:
Thanks for working on this! 

I'm looking at updating the string and a couple of semi-related issues in the latest posts block over in Gutenberg in https://github.com/WordPress/gutenberg/pull/55029/files#diff-55d79982c1f505132fcff0c29e1e84b411c56fbbf5dc1c55220a530125ebf8bdR489